docs/tasks/administer-federation/cluster.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Federated Cluster
+redirect_from:
+- "/docs/user-guide/federation/cluster/"
+- "/docs/user-guide/federation/cluster.html"
+---
+
+{% capture overview %}
+
+This guide explains how to use Clusters in Federation control plane.
+
+Different of other Kuberentes resources, such as Deployments, Services and ConfigMaps,
+clusters only exist in the federation context, i.e. those requests must be submitted to the
+federation api-server.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include federated-task-tutorial-prereqs.md %}
+* You should also have a basic [working knowledge of Kubernetes](/docs/setup/pick-right-solution/) in
+general.
+
+{% endcapture %}
+
+{% capture steps %}
```

Comment:
`{% capture steps %}` was not closed. Looks like you need an `{% endcapture %}` at the end of the section.